shared/constants/config.tsx

```diff
@@ -902,7 +902,8 @@ export const _useConfigState = Z.createZustand<State>((set, get) => {
         s.justRevokedSelf = ''
       })
     },
-    resetState: () => {
+    resetState: isDebug => {
+      if (isDebug) return
```

Comment:
if we reset this store it'll try and rebootstrap which we don't want while we're debugging